projects/m2k/common/m2k_bd.tcl

```diff
@@ -27,10 +27,9 @@ create_bd_port -dir O -from 11 -to 0 txd
 ad_ip_parameter sys_ps7 CONFIG.PCW_EN_CLK2_PORT 1
```

Comment:
If we don't use it, we should not enable it.

Reply:
Of course, thanks.